ocrmypdf is failing due to `PermissionError: [Errno 1] Operation not permitted` running one of its tests.

```
______________________ test_unpaper_args_invalid_filename ______________________
[gw5] darwin -- Python 3.10.4 /nix/store/9ds1mv2cpdii4nv56jszcsfgk0lc0r6m-python3-3.10.4/bin/python3.10
resources = PosixPath('/private/tmp/nix-build-python3.10-ocrmypdf-13.4.5.drv-0/source/tests/resources')
outpdf = PosixPath('/private/tmp/nix-build-python3.10-ocrmypdf-13.4.5.drv-0/pytest-of-_nixbld1/pytest-0/popen-gw5/test_unpaper_args_invalid_file0/out.pdf')

    @needs_unpaper
    def test_unpaper_args_invalid_filename(resources, outpdf):
        p = run_ocrmypdf(
            resources / "skew.pdf",
            outpdf,
            "-c",
            "--unpaper-args",
            "/etc/passwd",
            '--plugin',
            'tests/plugins/tesseract_noop.py',
        )
>       assert "No filenames allowed" in p.stderr
E       assert 'No filenames allowed' in 'Traceback (most recent call last):\n  File "/nix/store/9ds1mv2cpdii4nv56jszcsfgk0lc0r6m-python3-3.10.4/lib/python3.10...-packages/ocrmypdf/_>
E        +  where 'Traceback (most recent call last):\n  File "/nix/store/9ds1mv2cpdii4nv56jszcsfgk0lc0r6m-python3-3.10.4/lib/python3.10...-packages/ocrmypdf/__main__.py", line 38, i>

tests/test_unpaper.py:88: AssertionError
```

--- a/pkgs/development/python-modules/ocrmypdf/default.nix
+++ b/pkgs/development/python-modules/ocrmypdf/default.nix
@@ -2,6 +2,7 @@
 , buildPythonPackage
 , coloredlogs
 , fetchFromGitHub
+, fetchpatch
 , ghostscript
 , img2pdf
 , importlib-metadata
@@ -52,6 +53,11 @@ buildPythonPackage rec {
       pngquant = "${lib.getBin pngquant}/bin/pngquant";
       tesseract = "${lib.getBin tesseract4}/bin/tesseract";
       unpaper = "${lib.getBin unpaper}/bin/unpaper";
+    })
+    # https://github.com/ocrmypdf/OCRmyPDF/pull/973
+    (fetchpatch {
+      url = "https://github.com/ocrmypdf/OCRmyPDF/commit/808b24d59f5b541a335006aa6ea7cdc3c991adc0.patch";
+      hash = "sha256-khsH70fWk5fStf94wcRKKX7cCbgD69LtKkngJIqA3+w=";
     })
   ];
 

